lib/datadog/appsec/context.rb

```diff
@@ -36,7 +37,9 @@ def initialize(trace, span, waf_runner)
         @span = span
         @events = []
         @waf_runner = waf_runner
+
         @metrics = Metrics::Collector.new
+        @request_milestones = RequestMilestones.new
```

Comment:
I don't think that "request milestones" reflect the purpose, it's same metrics. Based on the description

> Data about the milestones that happened during request processing

Judging by the description of the class it's request processing stats. The issue with current name - it creates an expression of something more when it's actually just stats.

Reply:
I think you are right, "milestones" is not a good name here. But I think that "metrics" or "stats" also wouldn't work - these are more flags than metrics.